src/Servers/HttpSys/test/FunctionalTests/Http2Tests.cs

```diff
@@ -21,6 +21,7 @@
 
 namespace Microsoft.AspNetCore.Server.HttpSys.FunctionalTests;
 
+[SkipOnHelix("Unsupported queue", Queues = "Windows.Amd64.VS2022.Pre.Open;")]
```

Comment:
Why don't these tests run on this queue? Is HTTP.sys and IIS not on the image?

It would be good to have a better description in the attribute.

Reply:
Are all the HTTP.sys and IIS functional tests skipped? If so, SkipOnHelix attribute might work on the assembly. Avoids modifying so many files.

Reply:
Based on the usage attributes the SkipOnHelix attribute doesn't work at the assembly level. The reason I am disabling these tests is that they are causing failtures due to hitting some 40m threshold. I'm not certain if it's that IIS just won't work on that image or if it is something else at this stage.